Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/ForecastDescriptor.java

```diff
@@ -100,7 +100,7 @@ void setIntervalPrecision(float[] calibration) {
 
     @Deprecated
     float[] getCalibration() {
```

Comment:
getIntervalPrecision was already allowed via Lombok. Changed it explicitly. One reason for keeping the deprecated version was that this name was used in 3.7.0 and this would be a breaking change. But at the same time RCFCaster has had a revamp and the new name of the variable is more thought through. That being said it is unclear if this was in use somewhere --  we can add it back if there is an ask.